### PR TITLE
release v1

### PR DESCRIPTION
**Pull request by van-fs**
This PR readies the app for V1:
- Updated the readme with more detailed setup/use instructions
- Fixed a bug in the ProductService: it was picking the first value from the observable, which was an array, not the first value/product in the array
- Updated titles to Angular Shoppe similar to Reactshoppe
- Updated the CartComponent's title to include Your Cart; this also required a bit of shuffling on the Keep Shoppeing button
- Updated the CheckoutComponent to have IDs on elements and exclusion rules for better compatibility with FSTA
- Set the mock API to default out of the box
- Bump the version to 1.0.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,60 @@
...
+- `Changed` `CSS Selector` #firstname
+- `Visited URL` `is` `PATH` /thankyou
+
+Check out a few sessions and explore the data to get a feel for how you'd use FullStory on your own app or site.
+
+## Tips and tricks
+
+- If you want to simulate different users, use a private or incognito window with the app.
+- Use the [FullStory Browser SDK](https://github.com/fullstorydev/fullstory-browser-sdk) to easily load FullStory into your app or site. See `app.component.ts`.
+- Add `fs-exclude` CSS classes to prevent recording of sensitive or personal information. See `checkout.component.html`.
```
Comment (patrick-fs) [inline README.md]: A link to our updated privacy (PbD) controls page could go here. https://help.fullstory.com/hc/en-us/articles/360044349073-FullStory-Private-by-Default

File: README.md [hunk 2]
```diff
@@ -1,4 +1,60 @@
...
+Find your session or build a segment using the following event filters.
+
+- `Clicked` `Text is exactly` **Add to Cart**
+- `Visited URL` `is` `PATH` /cart
+- `Visited URL` `is` `PATH` /checkout
+- `Changed` `CSS Selector` #firstname
+- `Visited URL` `is` `PATH` /thankyou
+
+Check out a few sessions and explore the data to get a feel for how you'd use FullStory on your own app or site.
+
```
Comment (patrick-fs) [inline README.md:45]: It would be cool if you can create an animated gif of one of those segments being created.

**Review by patrick-fs — APPROVED** [review]
LGTM! I left a few comments for you to review about the Readme.

File: README.md (hunk 2 above)
Comment (van-fs) [reply README.md:45]: Agree.  I was trying not to add too much to this PR ;) ... will mark this as an issue for future PR.

**Pull request merged**